examples/meta/src/evaluation/cross_validation_linear_svm.sg

```diff
@@ -0,0 +1,38 @@
+CSVFile f_feats_train("../../data/classifier_binary_2d_linear_features_train.dat")
+CSVFile f_labels_train("../../data/classifier_binary_2d_linear_labels_train.dat")
+
+#![create_features]
+RealFeatures combined_features(f_feats_train)
```

Comment:
I wouldnt use combined features, this is more multiple kernel learning

Reply:
Changed them to features_train and features_test respectively.